js/web/lib/wasm/wasm-core-impl.ts

```diff
@@ -114,6 +114,8 @@ export const initEp = async (env: Env, epName: string): Promise<void> => {
   if (!BUILD_DEFS.DISABLE_JSEP) {
     // eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires
     const initJsep = require('./jsep/init').init;
+    const wasm = getInstance();
+    wasm.traceEvent = env.trace;
```

Comment:
This should not be assigned in this context. Please modify onnxruntime\wasm\pre-jsep.js to assign `Module["traceEvent"]` there.